### DISCUSSION TITLE
HD300 Died!  Any schematics?? (solved with new buck)

### DISCUSSION THREAD

**Discussion by jbuszkie** [OpenSageTV/Community#116] [SageTV Extenders (HD300,HD200)]
Hello!  My wife somehow killed our HD300. (It really may not have been anything to do with her! lol)  The power adapter is fine and I'm getting 12V to the circuit board.  But nothing on the first power supply on the circuit board.  I get no power LED light either.  Are there schematics floating around for the HD300?  I'm trying to see if there is something in can fix.  The schematics would help alot!

**Comment by TGsell1** [comment]
Try power cycling by repeatedly plugging in the power jack until you get the power LED to light and then stop and let it boot.  There are times it’s taken me 20 or 30 times but it has worked.  And then I leave it on a UPS so a power glitch doesn’t kill it.  Another trick I have found is to do the above with HDMI plugged into a monitor vs a TV as they provide a different voltage profile to the HD300.  I wish I knew why any of these things worked but at this point I have recovered one or two this way I thought were dead.  Sent from my iPhoneOn Feb 25, 2024, at 9:38 AM, jbuszkie ***@***.***> wrote:﻿
Hello!  My wife somehow killed our HD300. (It really may not have been anything to do with her! lol)  The power adapter is fine and I'm getting 12V to the circuit board.  But nothing on the first power supply on the circuit board.  I get no power LED light either.  Are there schematics floating around for the HD300?  I'm trying to see if there is something in can fix.  The schematics would help alot!

**Comment by jbuszkie** [comment]
I tried that with no luck..   I bypassed the first internal switching power supply and supplied 5V on that output and the blue light came on.  So it seems there something wrong with the first power supply.  That IC isn't available anymore so it it's the IC I will have to hack it anyway..  More investigating to continue...
It's a shame that I might have just voided my warrantee by taking them apart!!  🤣

**Comment by TGsell1** [comment]
Wow you are far more sophisticated than I am with this device.... keep in touch on your investigation!

**Comment by jbuszkie** [comment]
I'm such a hack!!
![image](https://github.com/OpenSageTV/Community/assets/11888706/774122bb-a58f-4350-9607-1d7a9ac63373)
It worked though!!  We'll see how long it lasts..
![image](https://github.com/OpenSageTV/Community/assets/11888706/4c69b012-5ca7-45b7-9df8-a4badea1a9d3)

**Reply by trk2** [reply]
For the love of god adjust your overscan settings. :)

**Comment by TGsell1** [comment]
Wow! Where did you get the part you subbed into the power supply? I’d like to be able to replicate! Sent from my iPhoneOn Feb 25, 2024, at 12:05 PM, jbuszkie ***@***.***> wrote:﻿
I'm such a hack!!
image.png (view on web)
It worked though!!  We'll see how long it lasts..
image.png (view on web)

**Comment by TGsell1** [comment]
I see where you spliced into the power jack but what board is that and where does that board link into on the HD300? Sent from my iPhoneOn Feb 25, 2024, at 12:05 PM, jbuszkie ***@***.***> wrote:﻿
I'm such a hack!!
image.png (view on web)
It worked though!!  We'll see how long it lasts..
image.png (view on web)

**Comment by jbuszkie** [comment]
I got the part from Aliexpress years ago..  I have no idea if it's still available.  Any buck *should* work (original, on board one,  was rated for 2A),  As long as it fits and is about 2A, you should be ok.
I de-soldered the inductor and took off the diode and cap and lifted the Vin pin to the original switcher IC.  In retrospect I could have left the cap on,  I thought I might need the space.  But after looking back and the photo, I could have left it on,
I reality you only probably need to life the Vin pin of the IC and can probably leave the other components... but the safest would be to remove the inductor too.

The Vout of my new buck is then connected to the inductor pad.
![image](https://github.com/OpenSageTV/Community/assets/11888706/36e05fe8-bb31-4239-a98d-d51ba09e1a76)

**Comment by TGsell1** [comment]
I found these on Amazon which look like they might work as you suggest.  I’m handy but not like you,,, I get rhetorical Vin + and - input to the buck that’s easy.  When you lift the IC pin you connect the Vout + from the buck ? Where does the Vout - from the buck solder to?  I bet this will save a lot of HD300s!! Thanks!Sent from my iPhoneOn Feb 25, 2024, at 9:48 PM, jbuszkie ***@***.***> wrote:﻿
I got the part from Aliexpress years ago..  I have no idea if it's still available.  Any buck should work (original, on board one,  was rated for 2A),  As long as it fits and is about 2A, you should be ok.
I de-soldered the inductor and took off the diode and cap and lifted the Vin pin to the original switcher IC.  In retrospect I could have left the cap on,  I thought I might need the space.  But after looking back and the photo, I could have left it on,
I reality you only probably need to life the Vin pin of the IC and can probably leave the other components... but the safest would be to remove the inductor too.
The Vout of my new buck is then connected to the inductor pad.
image.png (view on web)

**Reply by jbuszkie** [reply]
> Where does the Vout - from the buck solder to?

On My unit the Vin- and Vout-  were connected together on the module.  So I didn't bother to connect Vout-...  But to be complete you may want to connect Vout- to the GND plane somewhere.

**Comment by TGsell1** [comment]
Got it.  Thanks Sent from my iPhoneOn Feb 25, 2024, at 11:36 PM, jbuszkie ***@***.***> wrote:﻿

Where does the Vout - from the buck solder to?
On My unit the Vin- and Vout-  were connected together on the module.  So I didn't bother to connect Vout-...  But to be complete you may want to connect Vout- to the GND plane somewhere.

**Comment by TGsell1** [comment]
I see now where you soldered the new  Vout to the left side of the inductor pad.  How did you know which side to solder to?  I have also seen on Amazon a buck similar to the original one with 8 pins.   That might be eventually be worth exploring but since you have a solution I wouldn’t mess with it  - I have an HD300 which has had the ‘no power’ symptom but is now working if it fails again I’m going to try this.   Did you have to adjust the output voltage at all?  This is a really cool solution! Thanks again Sent from my iPhoneOn Feb 25, 2024, at 9:48 PM, jbuszkie ***@***.***> wrote:﻿
I got the part from Aliexpress years ago..  I have no idea if it's still available.  Any buck should work (original, on board one,  was rated for 2A),  As long as it fits and is about 2A, you should be ok.
I de-soldered the inductor and took off the diode and cap and lifted the Vin pin to the original switcher IC.  In retrospect I could have left the cap on,  I thought I might need the space.  But after looking back and the photo, I could have left it on,
I reality you only probably need to life the Vin pin of the IC and can probably leave the other components... but the safest would be to remove the inductor too.
The Vout of my new buck is then connected to the inductor pad.
image.png (view on web)

**Reply by jbuszkie** [reply]
For some reason I'm not seeing any pictures you attach.  I'm looking directly on the github website.

**Reply by jbuszkie** [reply]
> How did you know which side to solder to?

I traced the right side went to the switcher IC.  I was able to pull a reference schematic off of the datasheet for that IC

**Comment by TGsell1** [comment]
I have another HD300 with non functioning Ethernet port.   Keith helped me get that one working with a wireless dongle.  Wondering if the port is a module that can be replaced? Sent from my iPhoneOn Feb 27, 2024, at 2:26 PM, jbuszkie ***@***.***> wrote:﻿
I don't know...  I'd have to look at the circuit board again to see if the 12V goes anywhere else.
I could be that it was cheaper to have a 12V, current wall wart than a 2A 5V wall wart???

**Comment by TGsell1** [comment]
Gotcha.   I found the Ethernet controller for $3 but I won’t be able to solder it! Sent from my iPhoneOn Feb 27, 2024, at 5:13 PM, jbuszkie ***@***.***> wrote:﻿

How did you know which side to solder to?

I traced the right side went to the switcher IC.  I was able to pull a reference schematic off of the datasheet for that IC

**Comment by TGsell1** [comment]
From your diagram it looks like your new Vout connects to the LEFT side of the inductor pad ?? Sorry for being thick headed.  Sent from my iPhoneOn Feb 27, 2024, at 5:13 PM, jbuszkie ***@***.***> wrote:﻿

How did you know which side to solder to?

I traced the right side went to the switcher IC.  I was able to pull a reference schematic off of the datasheet for that IC

**Comment by TGsell1** [comment]
must be something Im doing wrong with the attachments...  not worth investigating it was just an amazon pic
